packages/tracking/src/class-tracking.php

```diff
@@ -104,7 +104,10 @@ public function tracks_record_event( $user, $event_name, $properties = array(),
 		}
 		$terms_of_service = new Terms_Of_Service();
 		// Don't track users who have opted out or not agreed to our TOS, or are not running an active Jetpack.
-		if ( ! $terms_of_service->has_agreed() ) {
+		if (
+			! $this->connection->is_user_connected()
```

Comment:
Hey @zinigor ! Could you please elaborate on why you chose to use `is_user_connected` instead of `is_active` which was previously used in the terms-of-service package `has_agreed` method?
Thanks!

Reply:
That's a good point, this requires some explanation. I will add a part about that in the PR description now.